docs/changelog.rst

```diff
@@ -100,6 +100,7 @@ Other changes:
   case is shown on separate lines.
 * Refactored library.py file by splitting it into multiple modules within the
   beets/library directory.
+* :doc:`/docs/guides/main.rst`: Add instructions to install beets on Void Linux.
```

Comment:
You may need to remove the `/docs/` prefix here for the changelog to render correctly in the documentation.

You can check this locally by building the docs `poe docs` and having a look a the changelog page in the generated html file.

Reply:
I’m not too fond of it either 😆 it’s definitely a bit old-school. If all else fails and you can’t get it to work, I can jump in.